tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/AbstractPortalBundleFactory.java

```diff
@@ -112,9 +112,8 @@ private boolean detectLiferayHome( IPath path )
             return false;
         }
 
-        if( path.append( "osgi/core" ).toFile().exists() &&
-            path.append( "osgi/modules" ).toFile().exists() &&
-            path.append( "osgi/portal" ).toFile().exists() )
+        if( path.append( "data" ).toFile().exists() &&
```

Comment:
Data doesn't necessaryily have to exist.  Also previously the data folder didn't have to exist in GA1 we need this to support both GA1 and GA2